Formula/a/aptos.rb

```diff
@@ -37,9 +37,7 @@ class Aptos < Formula
   end
 
   def install
-    # FIXME: Look into a different way to specify extra RUSTFLAGS in superenv as they override .cargo/config.toml
-    # Ref: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/extend/ENV/super.rb#L65
-    ENV.append "RUSTFLAGS", "--cfg tokio_unstable -C force-frame-pointers=yes -C force-unwind-tables=yes"
+    ENV.append_to_rustflags "--cfg tokio_unstable -C force-frame-pointers=yes -C force-unwind-tables=yes"
 
```

Comment:
```suggestion
```
This can just be removed, since this should be `RUSTFLAGS` set by `.cargo/config.toml`.